test/ekka_autocluster_SUITE.erl

```diff
@@ -226,6 +241,53 @@ reboot_ekka_with_mcast_env() ->
     ok = set_app_env(node(), {mcast, ?MCAST_OPTIONS}),
     ok = ekka:start().
 
+t_autocluster_mcast_lock_failure(_Config) ->
+    ok = reboot_ekka_with_mcast_env(),
+    ok = timer:sleep(1000),
+    N1 = ekka_ct:start_slave(ekka, n1),
+    try
+        ok = ekka_ct:wait_running(N1),
+        ok = set_app_env(N1, {mcast, ?MCAST_OPTIONS})
+    after
```

Comment:
Q: is this missing some assertion? Maybe `assert_unlocked(ekka_cluster_mcast, N1)`?

Reply:
Thanks, fixed.